caravel/requirements/requirements-all.txt

```diff
@@ -0,0 +1,5 @@
+flask
+divvy
+#looper>=0.11.0
```

Comment:
Should these be uncommented now or are they intended to be silenced?

Reply:
Oh yeah, I've accidentally left it commented